core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -508,7 +582,7 @@ class TownsFundRoundFourValidationFailure(ValidationFailure):
     section: str
     column: str
     message: str
-    row_indexes: list[int]
+    row_indexes: list[int | Hashable]
```

Comment:
Why can row_indexes now be a Hashable data type? Don't we want consistency across all row_indexes?

Reply:
Nice spot

I made this change without thinking because my IDE says that we're returning `Hashable` because `iterrows` documentation says that. An `int` is a `Hashable` though, the IDE just isn't that clever.

I'll revert